Mathlib/Analysis/Analytic/OfScalars.lean

```diff
@@ -268,7 +268,7 @@ theorem ofScalars_radius_eq_zero_of_tendsto [NormOneClass E]
   rw [← coe_zero, coe_le_coe]
   have := FormalMultilinearSeries.summable_norm_mul_pow _ hr
   contrapose! this
-  apply not_summable_of_ratio_norm_eventually_ge ENNReal.one_lt_two
+  refine not_summable_of_ratio_norm_eventually_ge (r := 2) (by simp) ?_ ?_
```

Comment:
Is this change necessary for the PR?

Reply:
No, it means that the previous proof was abusing defeq between `(1 : Real) < (2 : Real)` and `(1 : ENNReal) < (2 : ENNReal)`.

Reply:
Ah got it! Then this is a good change. 👍